src/main/resources/application.yaml

```diff
@@ -45,6 +45,16 @@ idam:
     url: ${S2S_URL:http://localhost:4552}
     secret: ${S2S_SECRET:AAAAAAAAAAAAAAAA}
     name: ${S2S_NAME:bulk_scan_payments_processor}
+  api:
+    url: ${IDAM_API_URL}
+  client:
+    id: 'bsp'
+    secret: ${IDAM_CLIENT_SECRET}
+    redirect_uri: ${IDAM_CLIENT_REDIRECT_URI}
+  users:
+    bulkscan:
+      username: ${IDAM_USERS_BULKSCAN_USERNAME:bulkscan+ccd@gmail.com}
+      password: ${IDAM_USERS_BULKSCAN_PASSWORD:Password12}
```

Comment:
If we can remove default that helps reason is if you don't set up env vars in any particular environment application boot fails and it is easy to isolate the issue other wise default would be set and might take a while when tests fails.

Reply:
That's the approach I started with (didn't include any user), but got some mapping errors at application startup. I'll check what happens if I remove `users` root element from here - previously I had it included (and empty).